lib/portlayer/exec/commit.go

```diff
@@ -35,6 +35,13 @@ func Commit(ctx context.Context, sess *session.Session, h *Handle, waitTime *int
 	defer trace.End(trace.Begin(h.ExecConfig.ID))
 
 	c := Containers.Container(h.ExecConfig.ID)
+
+	// check for an in memory container migration and reject the commit if it is the case.
+	if c.ExecConfig.Migrated {
```

Comment:
cannot bail out at this point, cause the vm operation is still allowed in this case.